### DISCUSSION TITLE
v1.3.2 MSI file is detected as Win32/Wacapew.C!ml by Windows Defender

### DISCUSSION THREAD

**Discussion by gregsskyles** [mesonbuild/meson#12858] [General]
I tried to download the 1.3.2 MSI file, and was blocked by Windows Defender.
The detection points to this information-free page: https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Program%3AWin32%2FWacapew.C!ml&threatid=265744
Not sure there's anything that can be done; I'm guessing it's a false positive (the suffix of !ml suggests a machine learning algorithm perhaps). Have never tried to report a FP to Microsoft, can see if it's even possible.

**Comment by eli-schwartz** [comment]
Thanks for the alert. This seems to be https://github.com/pyinstaller/pyinstaller/issues/5668 but they didn't have any recommended solution either.

**Comment by bonzini** [comment]
This is (was) issue #13492.
